.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,74 @@
+name: native-metrics CI
+
+on:
+  push:
+    branches:
+    - open-source-main
```

Comment:
per slack convo, i don't personally care that we get multiple runs from PR's direclty in the repo. that won't happen for forked PRs. i think there's benefit in getting the runs against branches we push into this repo w/o having to create a PR. but open to what the team thinks?